oozie-to-airflow/README.md

```diff
@@ -163,20 +163,43 @@ $ ./init.sh
 ```
 
 You can run the program (minimally) by calling:
-`python o2a.py -i <INPUT_WORKFLOW_XML>`
+`python o2a.py -i <INPUT_WORKFLOW_FOLDER_PATH> -o <OUTPUT_FOLDER_PATH>`
 
-You can also specify the job.properties file, user, and output file.
+Example:
+`python o2a.py -i examples/demo -o output/demo`
 
-`python o2a.py -i <INPUT_FILE> -p <PROP_FILE> -u <USER> -o
-<OUTPUT_FILE>`
+This is the full usage guide, available by running `python o2a.py -h`
+
+```bash
```

Comment:
It's not a bash script. It's only a text about a bash script.

Reply:
Removed